account-kit/java/signer-sdk/src/test/java/StamperTests.java

```diff
@@ -23,30 +25,29 @@ public StamperTests() throws GeneralSecurityException {
     @BeforeEach
     public void setUp() throws GeneralSecurityException, InvalidProtocolBufferException {
         SecretBytes private_key = SecretBytes.copyFrom(
-                Hex.decode("dbb5689c93418eb95a0f54a9d56b4af289a00c80a0650c2463c8f9c0803cf52c"),
+                Hex.decode("32528601f4d9a10994b781bd2c1616d96d95b4b2e17116d7fc7003e8a86fc779"),
                 InsecureSecretKeyAccess.get());
         Bytes public_key = Bytes.copyFrom(Hex.decode(
-                "04d22099b7499f03055f540def34f76798bd28df87c111f3503715788f70ea57066b3b45d40ff0801e930d15959c0ba2ec22903685e9005e8add6d1f94c5d68bcb"));
+                "045e0e1f9e7761f87c02d947ab77363b38db79188962ddad15617d7b6e01b0e8f51bee4ca2e4c9ad26dbf9cec5008a83ca84ec7aabb820ada733ed46b4e221a76d"));
+
         HpkeParameters parameters = HpkeParameters.builder().setKemId(HpkeParameters.KemId.DHKEM_P256_HKDF_SHA256)
                 .setKdfId(HpkeParameters.KdfId.HKDF_SHA256).setAeadId(HpkeParameters.AeadId.AES_256_GCM)
                 .setVariant(HpkeParameters.Variant.NO_PREFIX).build();
         HpkePublicKey hpkePublicKey = HpkePublicKey.create(parameters, public_key, /* idRequirement= */null);
         HpkePrivateKey hpkePrivateKey = HpkePrivateKey.create(hpkePublicKey, private_key);
-        tekManager = TEKManager.InitializeTEKManagerFromHpkeKey(hpkePrivateKey);
+        tekManager = TekManager.fromHpkeKey(hpkePrivateKey);
```

Comment:
Are consumers of this SDK going to need to configure hpke parameters by hand like this test does? If so, we should really look for a way to simplify- this feels way too complicated to use. (Or if it's only needed for this test, then disregard).